src/validation-dispatcher/single-field-dispatcher.ts

```diff
@@ -0,0 +1,150 @@
+import {
+  FieldValidation,
+  ValidationResult,
+  createDefaultValidationResult,
+} from '../model';
+
+import {
+  arrayContainsEntries,
+  isFunction,
+  isLastIndexInArray,
+  isNorUndefinedOrNull,
+} from '../helper';
+
+interface ValidationParams {
```

Comment:
As we said, we could move this interface and the creator function to `validation-dispatcher.model.ts`

Reply:
moved

Reply:
maybe if it's local it could be even single-field-dispatcher.model.ts